pkg/oc/cli/describe/helpers.go

```diff
@@ -167,32 +167,18 @@ func webHooksDescribe(triggers []buildapi.BuildTriggerPolicy, name, namespace st
 	result := map[string][]DescribeWebhook{}
 
 	for _, trigger := range triggers {
-		var webHookTrigger string
 		var allowEnv *bool
 
 		switch trigger.Type {
-		case buildapi.GitHubWebHookBuildTriggerType:
-			webHookTrigger = trigger.GitHubWebHook.Secret
-
-		case buildapi.GitLabWebHookBuildTriggerType:
-			webHookTrigger = trigger.GitLabWebHook.Secret
-
-		case buildapi.BitbucketWebHookBuildTriggerType:
-			webHookTrigger = trigger.BitbucketWebHook.Secret
-
+		case buildapi.GitHubWebHookBuildTriggerType, buildapi.GitLabWebHookBuildTriggerType, buildapi.BitbucketWebHookBuildTriggerType:
 		case buildapi.GenericWebHookBuildTriggerType:
```

Comment:
Maybe change this to use reflect.TypeOf instead of a switch statement for what is basically one check?

Reply:
it's 4 checks.

Reply:
and it's not a type check.

Reply:
touché, i was somehow reading it as switch trigger.(type)
carry on

Reply:
yeah, well you fooled me into reading it that way too after your comment until i reread it :)